rest/views.py

```diff
@@ -465,3 +482,109 @@ def get_queryset(self):
         return StaticAsset.objects.filter(
             id=self.kwargs['sa_id']
         )
+
+
+class LearningResourceExportList(ListCreateAPIView):
+    """REST list view for exports"""
+
+    serializer_class = LearningResourceExportSerializer
+    permission_classes = (
+        ViewRepoPermission,
+        IsAuthenticated,
+    )
+
+    def get_queryset(self):
+        repo_slug = self.kwargs['repo_slug']
+        try:
+            exports = self.request.session[
+                EXPORTS_KEY][repo_slug]
+        except KeyError:
+            exports = []
+        return [{"id": lr_id} for lr_id in exports]
+
+    def create(self, request, *args, **kwargs):
+        try:
+            lr_id = int(request.data['id'])
+        except ValueError:
+            raise ValidationError("learning resource id must be a number")
+        repo_slug = self.kwargs['repo_slug']
+
+        learning_resource = get_resource(lr_id, self.request.user.id)
+        if learning_resource.course.repository.slug != repo_slug:
+            raise PermissionDenied()
+
+        if EXPORTS_KEY not in self.request.session:
+            self.request.session[EXPORTS_KEY] = {}
+
+        if repo_slug not in self.request.session[EXPORTS_KEY]:
+            self.request.session[EXPORTS_KEY][repo_slug] = []
+
+        exports = self.request.session[EXPORTS_KEY][repo_slug]
+        if lr_id not in exports:
+            exports.append(lr_id)
+            self.request.session.modified = True
+
+        url = reverse('learning-resource-export-detail',
+                      kwargs={'repo_slug': self.kwargs['repo_slug'],
+                              'lr_id': lr_id})
+        headers = {"Location": url}
+        return Response(
+            {"id": lr_id}, status=status.HTTP_201_CREATED, headers=headers)
+
+    # pylint: disable=unused-argument
+    def delete(self, request, *args, **kwargs):
+        """Delete all ids in session for this repository and user"""
+        try:
+            repo_slug = self.kwargs['repo_slug']
```

Comment:
Pretty minor: These two lines could just be:

```python
self.request.session[EXPORTS_KEY].pop(self.kwargs['repo_slug'], None)
```

I like to avoid a variable that's only used once. Also, this will prevent a `KeyError` (as long as `self.request.session[EXPORTS_KEY]` always exists).